app/scripts/controllers/newOverview.js

```diff
@@ -533,7 +537,8 @@ function OverviewController($scope,
   //   value: alerts object
   var updateDeploymentWarnings = function(deployment) {
     var notifications = {};
-
+    _.assign(notifications, ResourceAlertsService.getPausedDeploymentAlerts(deployment));
```

Comment:
Nit: I'd rather keep it simple here and use

```js
var notifications = ResourceAlertsService.getPausedDeploymentAlerts(deployment);
```

since you're not mixing together multiple objects.